base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleData.java

```diff
@@ -540,21 +541,37 @@ public void setCloudAppVehicleID(boolean cloudAppVehicleID){
     public Boolean getCloudAppVehicleID(){
         return getBoolean(KEY_CLOUD_APP_VEHICLE_ID);
     }
-
-    /**
-     * Sets a boolean value for OEM Custom VehicleData.
-     * @param vehicleDataName a String value
-     * @param vehicleDataState a boolean value
-     */
-    public void setOEMCustomVehicleData(String vehicleDataName, Boolean vehicleDataState){
-        setParameters(vehicleDataName, vehicleDataState);
-    }
-
-    /**
-     * Gets a boolean value for OEM Custom VehicleData.
-     * @return a Boolean value.
-     */
-    public Boolean getOEMCustomVehicleData(String vehicleDataName){
-        return getBoolean(vehicleDataName);
-    }
-}
+
+    /**
+     * Sets a boolean value for OEM Custom VehicleData.
+     * @param vehicleDataName a String value
+     * @param vehicleDataState a boolean value
+     */
+    public void setOEMCustomVehicleData(String vehicleDataName, Boolean vehicleDataState){
+        setParameters(vehicleDataName, vehicleDataState);
+    }
+
+    /**
+     * Gets a boolean value for OEM Custom VehicleData.
+     * @return a Boolean value.
+     */
+    public Boolean getOEMCustomVehicleData(String vehicleDataName){
+        return getBoolean(vehicleDataName);
+    }
+
+    /**
+     * Sets a boolean value for StabilityControlsStatus.
+     * @param stabilityControlsStatus a boolean value
+     */

```

Comment:
```suggestion
  /**
     * Sets the stabilityControlsStatus.
     *
     * @param stabilityControlsStatus See StabilityControlsStatus
     * @since SmartDeviceLink 7.0.0
     */


```